app/models/ext_management_system.rb

```diff
@@ -108,7 +108,8 @@ def hostname_uniqueness_valid?
            :port=,
            :security_protocol,
            :security_protocol=,
-           :to => :default_endpoint
+           :to => :default_endpoint,
+           :allow_nil => true
```

Comment:
this is unrelated to your changes, just handles a case when there is no endpoint, which can happen for orphan network manager